### PR TITLE
[DOCS] Improve windows build instruction via conda

### PR DESCRIPTION
**Pull request by tqchen**
Enables windows build with llvm support

### CODE REVIEW

**Comment by tqchen** [comment]
cc @tmoreau89 @jroesch @junrushao1994 @rkimball  @comaniac

**Review by comaniac — APPROVED** [review]
LGTM to the conda part.

File: docs/install/from_source.rst [hunk 1]
```diff
@@ -122,27 +122,56 @@ The configuration of TVM can be modified by `config.cmake`.
...
+
+Conda is a very handy way to the necessary obtain dependencies needed for running TVM.
+First, follow the `conda's installation guide <https://docs.conda.io/projects/conda/en/latest/user-guide/install/>`_
+to install miniconda or anaconda if you do not yet have conda in your system. Run the following command in a conda environment:
+
+.. code:: bash
+
+    conda env update -f conda/build-environment.yaml
+    conda activate tvm-build
+
```
Comment (junrushao1994) [inline docs/install/from_source.rst:138]: 1) It is better to create a new env with the yaml file you provide: https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#creating-an-environment-from-an-environment-yml-file
2) Document what these lines are doing

Comment (tqchen) [reply docs/install/from_source.rst:138]: I actually checked, conda env update create also creates the env(when the original name exists), and conda env create will error out.

**Pull request merged**